kbase-extension/static/kbase/js/common/jobCommChannel.js

```diff
@@ -78,29 +86,24 @@ define([
         STOP_UPDATE = 'stop_job_update',
         // messages to be sent to backend for job request types
         REQUESTS = {
-            // cancels the job
-            CANCEL: 'request-job-cancel',
-
+            // Cancels the job
+            CANCEL,
             // Requests status of all jobs in a cell or list of cells
             CELL_JOB_STATUS,
-
-            // Fetches info (not state) about a job, including the app id, name, and inputs.
-            INFO: 'request-job-info',
-
-            // Fetches job logs from kernel.
-            LOGS: 'request-job-log',
-
-            // retries the job
-            RETRY: 'request-job-retry',
-
+            // Fetches info (not state) about a job, including the app id, name, and inputs
+            INFO,
+            // Fetches job logs
+            LOGS,
+            // Retries the job
+            RETRY,
```

Comment:
unified and standardised all query names -- regardless of whether they are back or frontend. Hurrah!